### DISCUSSION TITLE
Idea on different type of licenses.

### DISCUSSION THREAD

**Discussion by dhanushnayak** [dashingsoft/pyarmor#1240] [Q&A]
What type of key is recommended if  one version  of code, needs to be deployed on different devices can we use obfuscation code in different devices

**Comment by dhanushnayak** [comment]
Can Pyarmor license can we used in Raspberry pi

**Comment by jondy** [comment]
Please check Pyarmor documentation

**Reply by dhanushnayak** [reply]
im facing **NameError: name '__pyarmor__' is not defined** on running pyarmor-7 obfuscate filename.py in RPI

**Reply by dhanushnayak** [reply]
if im try to obfuscate 2 or 3 files in RPI using pyarmor gen file.py  and pyarmor gen file1.py ,

Im facing`File "/home/pi/Testapp/code/dist/pyarmor_runtime_000000/__init__.py", line 12, in <module>
    raise ModuleNotFoundError('no pyarmor_runtime extension found')
ModuleNotFoundError: no pyarmor_runtime extension found`
